extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -144,6 +153,11 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
      * @private {boolean}
      */
     this.shouldSandbox_ = false;
+
+    /** @private {string} The random subdomain to load SafeFrame from */
+    this.safeFrameSubdomain_ = /** @type {string} **/ Services.cryptoFor(
```

Comment:
Why is type declaration necessary here?

Reply:
It was because getSecureRandomString's return value was {string|null}, but now that doesn't apply anymore